dear_petition/petition/utils.py

```diff
@@ -36,7 +38,9 @@ def make_datetime_aware(dt_str):
     if dt_str is None or dt_str == "":
         return None
     # a) Turn the datetime string into a timezone-naive datetime object
-    naive_datetime_obj = datetime.strptime(dt_str, DATETIME_FORMAT)
+    for fmt in DATETIME_FORMAT:
+        naive_datetime_obj = dateutil.parser.parse(dt_str)
+
```

Comment:
Why do we need to iterate through the `DATETIME_FORMAT` string in order to come up with the `naive_datetime_obj`? Can we remove line 41?

Reply:
Yes, good catch